<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nhạc</string>
  </property>
  <widget class="QFrame" name="musicFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPlainTextEdit" name="musicIdTxt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sửa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>50</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lưu</string>
    </property>
   </widget>
   <widget class="QLabel" name="musicIdlabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID bài hát</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="musicNameLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tên bài hát</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="musicNameTxt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="artistLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nghệ sĩ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="artistTxt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="thumbnailLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Thumbnail</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="thumbNailTxt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="importBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>801</width>
      <height>231</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>128</number>
    </attribute>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Artist</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Image</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Link</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Category_ID</string>
     </property>
    </column>
   </widget>
   <widget class="QPlainTextEdit" name="fileTxt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fileLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>File nhạc</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="categoryComboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Category</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="importBtn_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <zorder>musicIdTxt</zorder>
   <zorder>updateBtn</zorder>
   <zorder>saveBtn</zorder>
   <zorder>musicIdlabel</zorder>
   <zorder>musicNameLabel</zorder>
   <zorder>musicNameTxt</zorder>
   <zorder>artistLabel</zorder>
   <zorder>artistTxt</zorder>
   <zorder>thumbnailLabel</zorder>
   <zorder>thumbNailTxt</zorder>
   <zorder>tableWidget</zorder>
   <zorder>fileTxt</zorder>
   <zorder>fileLabel</zorder>
   <zorder>categoryComboBox</zorder>
   <zorder>importBtn</zorder>
   <zorder>importBtn_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
